tasks/create_containers.yml

```diff
@@ -18,8 +18,8 @@
           - /tmp
           - /run
           - /run/lock
-        volumes:
-          - /sys/fs/cgroup:/sys/fs/cgroup:ro
+        volumes: '{{ hostvars[host]["boot_docker_volumes"]|default([]) }} + {{ ["/sys/fs/cgroup:/sys/fs/cgroup:ro"] }}'
```

Comment:
I would prefer `'{{ hostvars[host]["boot_docker_volumes"]|default([]) + ["/sys/fs/cgroup:/sys/fs/cgroup:ro"] }}'` (not sure why `{{ ['aa'] }} + {{ ['bb'] }}` returns `['aa', 'bb']` instead of `['aa'] + ['bb']`)